plg_system_sendemail.php

```diff
@@ -0,0 +1,129 @@
+<?php
+/**
+ * @package     Plg_System_Tjlms
+ * @subpackage  Plg_System_Tjlms
+ *
+ * @author      Techjoomla <extensions@techjoomla.com>
+ * @copyright   Copyright (C) 2009 - 2018 Techjoomla. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+// No direct access.
+defined('_JEXEC') or die;
+
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\Factory;
+use Joomla\CMS\Session\Session;
+
+JHtml::_('jquery.token');
+
+$lang = JFactory::getLanguage();
+$lang->load('plg_system_sendemail', JPATH_ADMINISTRATOR);
+
+/**
+ * Plugin to send email in a bulk.
+ *
+ * @since  __DEPLOY_VERSION__
+ */
+class PlgSystemplg_System_Sendemail extends JPlugin
+{
+	/**
+	 * Constructor - Function used as a contructor
+	 *
+	 * @param   object  $subject  The object to observe
+	 * @param   array   $config   An array that holds the plugin configuration
+	 *
+	 * @retunr  class object
+	 *
+	 * @since  __DEPLOY_VERSION__
+	 */
+	public function __construct($subject, $config)
+	{
+		Text::script('PLG_SYSTEM_SENDEMAIL_BTN');
+		Text::script('PLG_SYSTEM_SENDEMAIL_SELECT_CHECKBOX');
+		Text::script('PLG_SYSTEM_SENDEMAIL_POPUP_HEADING');
+		Text::script('PLG_SYSTEM_SENDEMAIL_POPUP_EMAIL_SUBJECT');
+		Text::script('PLG_SYSTEM_SENDEMAIL_POPUP_EMAIL_BODY_MESSAGE');
+		Text::script('PLG_SYSTEM_SENDEMAIL_POPUP_SEND_BTN');
```

Comment:
@komalm I think we should load these js files only where this integration is needed. 

cc @coolbung

Reply:
Agree. This needs to be fixed before the release. 

We cannot have this js file loading on each and every page @komalm . It's a system plugin so you have to be extremely careful about what assets you load.